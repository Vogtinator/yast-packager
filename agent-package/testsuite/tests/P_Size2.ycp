{
return SCR::Execute(.setEnvironment,$["packageinfopath":"tests","language":"german","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return SCR::Execute(.setInstallSelection,["aaa_base", "aaa_dir", "aaa_skel"], false);
}
{
    return SCR::Read(.installSet);
}
{
    return SCR::Read(.diskSpace,[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]);
}
{
    return SCR::Read(.diskSpace,[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000]]);
}
{
    return SCR::Read(.diskSpace,[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000]]);
}
{
    return SCR::Read(.diskSpace,[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000]]);
}
{
    return SCR::Read(.diskSpace,[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000]]);
}
{
    return SCR::Read(.diskSpace,[$["name":"/","used":0,"free":10000]]);
}
{
    return SCR::Read(.diskSpace,[$["name":"bin","used":0,"free":10000]]);
}
{
    return SCR::Read(.diskSpace,[]);
}

