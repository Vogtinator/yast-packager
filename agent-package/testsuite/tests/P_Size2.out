(true)
(true)
([["a1/aaa_base.rpm", "SuSE Linux Basispaket", 1, true], ["a1/aaa_dir.rpm", "SuSE Linux Verzeichnisstruktur", 1, true], ["a1/aaa_skel.rpm", "SuSE Linux Default-Benutzer", 1, false], ["a1/glibc.rpm", "Die shared Libraries der C-Library", 1, true]])
([$["free":7323, "name":"/", "used":2677], $["free":9972, "name":"bin", "used":28], $["free":9316, "name":"etc", "used":684], $["free":9424, "name":"sbin", "used":576], $["free":-36330, "name":"usr", "used":46330], $["free":9970, "name":"var", "used":30]])
([$["free":7293, "name":"/", "used":2707], $["free":9972, "name":"bin", "used":28], $["free":9316, "name":"etc", "used":684], $["free":9424, "name":"sbin", "used":576], $["free":-36330, "name":"usr", "used":46330]])
([$["free":-39037, "name":"/", "used":49037], $["free":9972, "name":"bin", "used":28], $["free":9316, "name":"etc", "used":684], $["free":9424, "name":"sbin", "used":576]])
([$["free":-39613, "name":"/", "used":49613], $["free":9972, "name":"bin", "used":28], $["free":9316, "name":"etc", "used":684]])
([$["free":-40297, "name":"/", "used":50297], $["free":9972, "name":"bin", "used":28]])
([$["free":-40325, "name":"/", "used":50325]])
([$["free":9972, "name":"bin", "used":28]])
([])
